addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.js

```diff
@@ -17,6 +18,33 @@ import { DashboardSearchBar } from "./dashboard_search_bar/dashboard_search_bar"
 
 export const dashboardActionRegistry = new Registry();
 
+const LAST_OPENED_DASHBOARD_ID_KEY = "spreadsheet_dashboard.last_opened_dashboard_id";
+
+/**
+ * Store the last opened dashboard id in browser storage.
+ * @param {number} dashboardId - The ID of the dashboard to store.
+ */
+function saveLastDashboardId(dashboardId) {
```

Comment:
I'd add "localStorage" somewhere in the function name to make it very clear  where it's called what it does. Same for `getLastDashboardId`.

Btw, the docstring text would be useless if "localStorage" is in the name :wink: